pkg/kor/all.go

```diff
@@ -80,6 +80,15 @@ func getUnusedRoles(kubeClient *kubernetes.Clientset, namespace string) Resource
 	return namespaceSADiff
 }
 
+func getUnusedHpas(kubeClient *kubernetes.Clientset, namespace string) ResourceDiff {
+	hpaDiff, err := processNamespaceHpas(kubeClient, namespace)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Failed to get %s namespace %s: %v\n", "hpas", namespace, err)
+	}
+	namespaceSADiff := ResourceDiff{"HPA", hpaDiff}
```

Comment:
Change to lowercase to match the return from the singular json function
namespaceSADiff := ResourceDiff{"Hpa", hpaDiff}